katello-installer/katello-installer-base.spec

```diff
@@ -9,85 +9,75 @@ Summary: Puppet-based installer for the Katello and Katello Capsule
 Group:   Applications/System
 License: GPLv3+ and ASL 2.0
 URL:     http://katello.org
-Source0: https://github.com/Katello/katello-installer/archive/%{version}.tar.gz
+Source0: https://github.com/Katello/katello-installer/archive/katello-installer-%{version}.tar.gz
 
 BuildArch: noarch
-Conflicts: foreman-installer
 Obsoletes: katello-installer < 2.1.0
 
+Requires: foreman-installer
 Requires: %{?scl_prefix}puppet >= 3.4.0
-Requires: %{?scl_prefix}rubygem-kafo
-Requires: %{?scl_prefix}rubygem-apipie-bindings >= 0.0.6
-Requires: foreman-selinux
 Requires: katello-selinux
 Requires: openssl
 
-%package -n katello-installer
-Summary: Puppet-based installer for Katello Server
+%package -n foreman-installer-katello
+Summary: Katello scenario for foreman-installer
 Group:	 Applications/System
-Conflicts: capsule-installer
-Conflicts: katello-devel-installer
+Obsoletes: katello-installer
 Requires: %{name} = %{version}-%{release}
 
-%description -n katello-installer
+%description -n foreman-installer-katello
 A set of tools for installation of Katello and a the default Capsule.
 
-%files -n katello-installer
-%{_datadir}/katello-installer/bin
-%{_datadir}/katello-installer/checks
-%config %{_sysconfdir}/katello-installer/config_header.txt
-%dir %{_sysconfdir}/katello-installer
-%dir %{_localstatedir}/log/katello-installer
-%config %attr(600, root, root) %{_sysconfdir}/katello-installer/katello-installer.yaml
-%config %attr(600, root, root) %{_sysconfdir}/katello-installer/capsule-certs-generate.yaml
-%config(noreplace) %attr(600, root, root) %{_sysconfdir}/katello-installer/answers.katello-installer.yaml
-%config(noreplace) %attr(600, root, root) %{_sysconfdir}/katello-installer/answers.capsule-certs-generate.yaml
-%{_sbindir}/katello-installer
+%post -n foreman-installer-katello
+foreman-installer --scenario katello --migrations-only > /dev/null
+
+%files -n foreman-installer-katello
+%{_datadir}/foreman-installer-katello/bin
+%config(noreplace) %attr(600, root, root) %{_sysconfdir}/foreman-installer/scenarios.d/katello-answers.yaml
+%config(noreplace) %attr(600, root, root) %{_sysconfdir}/foreman-installer/scenarios.d/katello.yaml
+%dir %{_sysconfdir}/foreman-installer/scenarios.d/katello.migrations
+%{_sysconfdir}/foreman-installer/scenarios.d/katello.migrations
 %{_sbindir}/capsule-certs-generate
 %{_sbindir}/katello-certs-check
 
-%package -n capsule-installer
-Summary:   Puppet-based installer for a Katello Capsule
+%package -n foreman-installer-capsule
+Summary:   Capsule scenario for Foreman installer
```

Comment:
Update this line to be consistent with line 23 (e.g. use 'foreman-installer' or 'Foreman installer' for both)